packages/snaps-execution-environments/package.json

```diff
@@ -17,11 +17,11 @@
     "test:browser": "wdio run wdio.config.ts",
     "test:ci": "yarn test",
     "test:watch": "jest --watch",
-    "lint:eslint": "eslint . --cache --ext js,ts",
-    "lint:misc": "prettier '**/*.json' '**/*.md' '!CHANGELOG.md' --ignore-path .gitignore",
+    "lint:eslint": "eslint . --cache --ext js,ts,jsx,tsx",
+    "lint:misc": "prettier --no-error-on-unmatched-pattern --loglevel warn \"**/*.json\" \"**/*.md\" \"**/*.html\" \"!CHANGELOG.md\" --ignore-path ./.prettierignore",
```

Comment:
Why are we using a prettierignore file here instead of the gitignore?

Reply:
I had to add `lavamoat/**/*.json`, because otherwise the generated policy files are different.